ocm/sign.py

```diff
@@ -174,17 +218,24 @@ def normalise_component_descriptor(
     component_descriptor: ocm.ComponentDescriptor,
     component_descriptor_lookup: collections.abc.Callable[[ocm.ComponentIdentity], ocm.ComponentDescriptor], # noqa: E501
     access_to_digest_lookup: collections.abc.Callable[[ocm.Access], ocm.DigestSpec],
+    verify: bool=False,
     normalisation: ocm.NormalisationAlgorithm=ocm.NormalisationAlgorithm.JSON_NORMALISATION,
 ) -> list[dict]:
     '''
     Returns a normalised version of the component descriptor by dropping signing-irrelevant
     properties, calculating missing digests and converting properties to the format expected by the
     OCM-cli (i.e. extra-identity handling and datetime-format).
 
+    @param component_descriptor:
+        the component descriptor of which a normalised representation should be created
     @param component_descriptor_lookup:
         lookup for component descriptors by their identity
-    @param image_ref_to_digest_lookup:
-        used to retrieve the digest of an oci-resource by its image reference
+    @param access_to_digest_lookup:
+        used to retrieve the digest of an resource by its access specification
+    @param verify:
```

Comment:
nit: there are different kinds of verification (we might, e.g. also check compliance to json-schema..)
-> how about naming this `verify_digests` instead?